### ISSUE TITLE
aten.cusum

### ISSUE THREAD

**Issue by desertfire** [pytorch/torchdynamo#883]

**Comment by jansel** [comment]
I don't think it is currently possible to write this one in Triton, so this is blocked on Triton language features.

**Comment by lezcano** [comment]
What about doing a expand + `tril` + `sum`? It would do some extra computation but it unblock this + it would be parallelizable. The only issue is that we would need play around a bit with the dimensions as `tril` just works on matrices, and `cumsum` takes an optional `dim`.

**Comment by lezcano** [comment]
And you would be right. I will implement the multi-kernel approach from JAX, see if it gets a more reasonable speed.
